src/components/ProgramList.js

```diff
@@ -67,6 +69,9 @@ class ProgramList extends Component {
       }))
       .then(() => fetchPrograms({
         user__not: user.user_id,
+      }))
```

Comment:
Since a user should never be able to remove a featured program, this seems unnecessary. The same could be said for the `user__not` above

Reply:
Good point. Removing them both.

Reply:
Whoops, apparently they are in fact needed!